### PR TITLE
doc time_classify_day: add an explanation

### PR DESCRIPTION
**Pull request by komainu8**

### CODE REVIEW

File: doc/source/reference/functions/time_classify_day.rst [hunk 1]
```diff
@@ -8,11 +8,111 @@
 Summary
 -------
 
+``time_classify_day`` rounds date to a day unit.
```
Comment (kou) [inline doc/source/reference/functions/time_classify_day.rst:11]: "date" -> "time"

File: doc/source/reference/functions/time_classify_day.rst [hunk 2]
```diff
@@ -8,11 +8,111 @@
...
+
+That is rounded "2020-01-30 11:50:11.000000" and "2020-01-30 22:50:11.000000" to "2020-01-30 00:00:00.000000".
```
Comment (kou) [inline doc/source/reference/functions/time_classify_day.rst:13]: "This rounds ..."

Comment (kou) [reply doc/source/reference/functions/time_classify_day.rst:13]: `"2020-01-30 11:50:11.000000"` -> ``` ``2020-01-30 11:50:11.000000`` ```

File: doc/source/reference/functions/time_classify_day.rst [hunk 3]
```diff
@@ -8,11 +8,111 @@
...
+
 Syntax
 ------
 
+``time_classify_day`` has one parameters::
```
Comment (kou) [inline doc/source/reference/functions/time_classify_day.rst:18]: "parameter" (no "s")

File: doc/source/reference/functions/time_classify_day.rst [hunk 4]
```diff
@@ -8,11 +8,111 @@
...
+  #   ]
+  # ]
+
+Parameters
+----------
+
+Required parameters
+^^^^^^^^^^^^^^^^^^^
+
+There are three required parameters ``column``.
```
Comment (kou) [inline doc/source/reference/functions/time_classify_day.rst:106]: "one"

File: doc/source/reference/functions/time_classify_day.rst (hunk 1 above)
Comment (komainu8) [reply doc/source/reference/functions/time_classify_day.rst]: Thank you for your comment.
I've fixed it.

File: doc/source/reference/functions/time_classify_day.rst (hunk 2 above)
Comment (komainu8) [reply doc/source/reference/functions/time_classify_day.rst]: Thank you for your comment.
I've fixed it.

File: doc/source/reference/functions/time_classify_day.rst (hunk 3 above)
Comment (komainu8) [reply doc/source/reference/functions/time_classify_day.rst]: Thank you for your comment.
I've fixed it.

File: doc/source/reference/functions/time_classify_day.rst (hunk 4 above)
Comment (komainu8) [reply doc/source/reference/functions/time_classify_day.rst]: Thank you for your comment.
I've fixed it.

File: doc/source/reference/functions/time_classify_day.rst [hunk 5]
```diff
@@ -8,11 +8,111 @@
...
 -------
 
+``time_classify_day`` rounds time to a day unit.
+
+This rounded ``2020-01-30 11:50:11.000000`` and ``2020-01-30 22:50:11.000000`` to ``2020-01-30 00:00:00.000000``.
+
 Syntax
 ------
 
+``time_classify_day`` has one parameter::
```
Comment (kou) [inline doc/source/reference/functions/time_classify_day.rst:18]: Could you use "This function has only one parameter" to reuse existing translation?

File: doc/source/reference/functions/time_classify_day.rst [hunk 6]
```diff
@@ -8,11 +8,111 @@
...
+
+  load --table Sales
+  [
+  {"name": "Apple" , "price": "256", "timestamp": "2020-05-01 11:50:11.000000"},
+  {"name": "Apple" , "price": "256", "timestamp": "2020-05-01 10:20:00.000000"},
+  {"name": "Orange", "price": "122", "timestamp": "2020-05-01 11:44:12.000001"},
+  {"name": "Apple" , "price": "256", "timestamp": "2020-05-01 19:50:23.000020"},
+  {"name": "Banana", "price": "88" , "timestamp": "2020-05-01 11:00:02.000000"},
+  {"name": "Banana", "price": "88" , "timestamp": "2020-05-01 21:34:12.000001"}
+  ]
```
Comment (kou) [inline doc/source/reference/functions/time_classify_day.rst:44]: Could you use `.. groonga-command`?

File: doc/source/reference/functions/time_classify_day.rst [hunk 7]
```diff
@@ -8,11 +8,111 @@
...
+  #   ]
+  # ]
+
+Parameters
+----------
+
+Required parameters
+^^^^^^^^^^^^^^^^^^^
+
+There is one required parameter ``column``.
```
Comment (kou) [inline doc/source/reference/functions/time_classify_day.rst:106]: Could you use "There is only one required parameter." to reuse existing translation?

**Comment by komainu8** [comment]
@kou Thank you for your comments.
I've fixed them.

File: doc/locale/ja/LC_MESSAGES/reference.po [hunk 8]
```diff
@@ -30176,3 +30198,13 @@ msgstr ""
...
+
+#~ msgid "There is one required parameter ``column``."
+#~ msgstr "必須引数が一つあり、それは ``column`` です。"
+
+#~ msgid "Here is query to show round value of ``timestamp`` to a day unit."
+#~ msgstr ""
+#~ "これは、 ``timestamp`` の値を日単位に丸めた値を表示するクエリーです。"
+
+#~ msgid "``time_classify_day`` has one parameter::"
+#~ msgstr "``time_classify_day`` の引数は1つだけです::"
```
Comment (kou) [inline doc/locale/ja/LC_MESSAGES/reference.po:30210]: Could you remove unused translations?

Comment (komainu8) [reply doc/locale/ja/LC_MESSAGES/reference.po]: Ah...I had overlooked them.
I've removed them.

**Pull request merged**